caffeinated/admin/new/pricing/espresso_events_Pricing_Hooks.class.php

```diff
@@ -986,6 +986,14 @@ public function autosave_handling(Events_Admin_Page $event_admin_obj)
      */
     public function pricing_metabox()
     {
+        /**
+         * Fires before any logic runs for the "Event Tickets and Datetimes" metabox in the Event Espresso Event Editor
+         *
+         * @since $VID:$
+         * @param espresso_events_Pricing_Hooks $this
+         */
+        do_action('AHEE__espresso_events_Pricing_Hooks___pricing_metabox__start', $this);
```

Comment:
I'm guessing this was needed for the example add-on you did?  Even so, I think this should have been done in a branch off master and merged not here.

Reply:
Ya you're right and in most cases that would be the thing to do, but this will not conflict with anything and we haven't had the need for a hook there in however many years, so this is fine as is. I've got to get back into high gear with REM so going to cut corners where there is no need to do everything by the book.